images/kube-proxy/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM registry.svc.ci.openshift.org/ocp/4.2:hyperkube AS builder
+RUN mkdir -p /tmp/build; \
+    cp /usr/bin/hyperkube /tmp/build/hyperkube
```

Comment:
The eventual goal was for the binary to be built from the sdn tree (so that origin doesn't have to keep kube-proxy-related upstream patches). I don't know if @deads2k had wanted that to be done right away or if this is an OK first step...

Reply:
Right, I'll follow up with him.  That's what I thought initially, also.  Reverse-engineering how that's built in origin was making my head spin.  I'll find out/fix it up.

Reply:
what we've landed on is building `vendor/k8s.io/kubernetes/cmd/kube-proxy`  so where we used to have the cmd `hyperkube kube-proxy` it will now be `kube-proxy`.  Eliminating the need to build hyperkube saves a dependency on the origin patches to kubernetes, from what I understand (since sdn carries its own differnent patches to kubernetes).